src/components/AddComment.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
+import { postComment } from "../utils/API";
+
+const AddComment = ({ username, comments, setComments }) => {
+  const [newComment, setnewComment] = useState("");
+  const { review_id } = useParams();
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    postComment(review_id, "weegembump", newComment)
```

Comment:
This is where you could instead provide the logged in user

Reply:
You can currently post an empty string as the body of the comment - you can either:
- Validate the input to ensure text is present
- Add a `required` attribute to the textarea
- Add back end behaviour (this would be a future endeavour)